message_ix/tests/test_tutorials.py

```diff
@@ -49,6 +52,15 @@
 # Short, readable IDs for the tests
 ids = [arg[0][-1] for arg in tutorials]
 
+# List of data files required to run tutorials
+data_files = [
+    "westeroes_baseline_demand.xlsx",
```

Comment:
Can we please remove the extra "e" in "Westeroes" to be consistent with the rest?

Reply:
resolved

Reply:
Please click the "Resolve conversation button"!